### PR TITLE
Feat mitigate to oui style variables

### LINKED ISSUE #217: Validate OUI theme compliance - phase 1
This is a tracking issue for phase 1 of OUI theme plugin compliance, which will involve changes to OUI typography and color SASS variables (for both dark and light mode). **For full details, including dates and tooling, see https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4291**

If your plugin has no UI views or components, please add a comment to that effect, and we'll remove the repo from future look and feel update campaigns.

## Actions required

- [x] Acknowledge the issue by adding an assignee who will serve as a point-of-contact
- [x] Provide screenshots and navigation instructions for important views and screens in your plugin or application. This will assist with both manual and automatic validation of compliance
- [ ] Identify all typographic properties and styles
- [ ] Create issues to mitigate typographic styles not inherited from OUI components or SASS variables
- [ ] Identify all color values, including visualization colors and palettes
- [ ] Create issues to mitigate typographic styles not inherited from OUI components or SASS variables
- [ ] Resolve all mitigation issues
- [ ] Validate and test plugin with updated OUI theme

## Questions?

Add a comment on https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4291

### PR DESCRIPTION
**Pull request by wanglam**
### Description
This PR will remove all exists customized color and font style. Make sure all styles from oui or oui component. 
This PR will fix #217 

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ruanyl — APPROVED** [review]
Nice changes, approved!

File: public/components/monitoring/model_deployment_table.tsx [hunk 1]
```diff
@@ -98,8 +98,7 @@ export const ModelDeploymentTable = ({
             return (
               <EuiHealth className="ml-modelStatusCell" color="danger">
                 <div className="eui-textTruncate">
-                  <span style={{ fontWeight: 600 }}>Not responding</span> on {planningNodesCount} of{' '}
-                  {planningNodesCount} nodes
+                  <b>Not responding</b> on {planningNodesCount} of {planningNodesCount} nodes
```
Comment (joshuarrrr) [inline public/components/monitoring/model_deployment_table.tsx:101]: nit - I generally think we'd recommend `<strong>` over `<b>` for semantic purposes, but realistically t shouldn't matter.

File: public/ace-themes/sql_console.js [hunk 2]
```diff
@@ -1,19 +0,0 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import * as ace from 'brace';
-
-ace.define('ace/theme/sql_console', ['require', 'exports', 'module', 'ace/lib/dom'], function (
```
Comment (joshuarrrr) [inline public/ace-themes/sql_console.js:8]: I may have missed in this PR where you were previously specifying use of this custom `ace` theme, but for consistency with OpenSearch Dashboards (particularly in dark mode), you should use the `textmate` theme in any ace/EuiCodeEditor component. See https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4609 for an example of similar cleanup in core.

**Review by joshuarrrr — APPROVED** [review]
From the technical side, all these changes look great, thanks! Make sure to get final screenshots or demos validated by UX.

File: public/ace-themes/sql_console.js (hunk 2 above)
Comment (wanglam) [reply public/ace-themes/sql_console.js:8]: Sure. Thank you. The code editor was been used in `feature/model-registry` branch. It's not been used in current application. Is that means we don't need to provide more scss files by using `textmate` theme?

**Review by ruanyl — APPROVED** [review]

**Pull request merged**